src/metadata/metadata.c

```diff
@@ -1649,12 +1646,11 @@ void ocf_metadata_load_properties(ocf_volume_t volume,
 	int result;
 
 	OCF_DEBUG_TRACE(volume->cache);
-	
+
 	result = ocf_metadata_read_sb(volume->cache->owner, volume,
 			ocf_metadata_load_properties_cmpl, cmpl, priv);
-	if (result) {
+	if (result)
 		OCF_CMPL_RET(priv, result, NULL);
-	}
 }
```

Comment:
Here you revert the change made in one of the previous commits. Please clean this up.